docs/getting-started-locally.md

```diff
@@ -0,0 +1,63 @@
+# Starting an Interledger Node Locally Using moneyd
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive value using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+This tutorial is useful if you’re doing local development. 
+
+![Getting-Started-Locally](../assets/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
+* Set up Node to install modules globally <br/>
+Open a command line and use the following commands:  
```

Comment:
Just to clarify, we'll remove this section?

Reply:
Yes, I was thinking of. Do you want to keep it?

Reply:
No, I think we should remove it, but only because it wasn't necessary for my installation. I will defer to the JS experts.

Reply:
Just confirming, setting up Node to install modules globally isn't necessary or the steps on how to install modules globally isn't necessary?

Reply:
Well, I was referring to lines 15-18 -- this is not really "enabling global installation" so much. Instead, it's changing npm's default directory to another directory. In my case, I already had a default directory working fine, so executing this command wasn't necessary.

I guess my overall point is that this tutorial should _not_ go into very much details around NodeJS configuration. There's already a lot of content there available in other places on the web.